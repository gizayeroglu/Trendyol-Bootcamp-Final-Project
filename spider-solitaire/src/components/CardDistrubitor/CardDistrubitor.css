.card-distrubitor-container{
    display: flex;
    align-items:flex-end;
    justify-content: flex-end;
    margin: 50px 30px;
}
