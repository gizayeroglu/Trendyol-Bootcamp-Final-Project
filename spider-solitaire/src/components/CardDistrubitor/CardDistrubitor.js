import React from 'react';
import Card from '../Card/Card';
import './CardDistrubitor.css';

function CardDistrubitor(){
  return (
    <>   
      <div className="card-distrubitor-container">
      <Card />
      </div>
    </>
  );
}

export default CardDistrubitor;