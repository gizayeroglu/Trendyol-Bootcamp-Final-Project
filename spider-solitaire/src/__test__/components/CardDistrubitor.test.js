import React from "react";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import { configure, shallow, mount } from "enzyme";

import CardDistrubitor from "../../components/CardDistrubitor/CardDistrubitor";

configure({ adapter: new Adapter() });

describe("Card Distrubitor component tests", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CardDistrubitor />);
  });

  it("should render the card holder component", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should have a div with the className card-distrubitor-container", () => {
    let div = wrapper.find("div");
    expect(div.hasClass("card-distrubitor-container")).toBeTruthy();
  });
});
