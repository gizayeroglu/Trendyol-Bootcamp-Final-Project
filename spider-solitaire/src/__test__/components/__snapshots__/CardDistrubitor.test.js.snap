// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Distrubitor component tests should render the card holder component 1`] = `ShallowWrapper {}`;
